internal/integrationtest/util_test.go

```diff
@@ -64,7 +64,7 @@ var http2spec = integrationtest.TransportSpec{
 		return http.NewTransport()
 	},
 	NewUnaryOutbound: func(x peer.Transport, pc peer.Chooser) transport.UnaryOutbound {
-		return x.(*http.Transport).NewOutbound(pc)
+		return x.(*http.Transport).NewOutbound(pc, http.EnableHTTP2())
```

Comment:
nit: should we call it "useHTTP2"? For me "enable HTTP2" mean "use http2 if possible", and "use http2" is "use http2 unconditionally", i.e. the latter would fail if server doesn't accept http2 requests.

Basically, it's also a question: how the http2 client works, would it fail if server can't accept http2 requests?